### PR TITLE
multiline pipeline params: remove max height for multilined fields

### PR DESCRIPTION
**Pull request by abhinandan13jan**
# Addresses
https://issues.redhat.com/browse/ODC-5754

# Screenshot
Screenshot from 2021-04-29 16-52-27

# Test
no changes

# Browser conformance
Chrome

### CODE REVIEW

File: frontend/packages/console-shared/src/components/formik-fields/field-types.ts [hunk 1]
```diff
@@ -97,6 +97,7 @@ export interface MultiColumnFieldProps extends FieldProps {
   emptyMessage?: string;
   headers: ({ name: string; required: boolean } | string)[];
   complexFields?: boolean[];
+  isMultiline?: boolean;
```
Comment (andrewballantyne) [inline frontend/packages/console-shared/src/components/formik-fields/field-types.ts:100]: As discussed on slack, but for posterity, this is not a valid solution. Almost all our usages of MultiColumnField are single lines suffering from this height issue. This prop would effectively always need to be on for every usage. 

As noted in the comments (and in Slack DMs) aim to see about removing the offending css and see if everything works fine. It's unclear why it was added but it seems to be an issue now (possible PF update of spacing to cause this issue)

Comment (abhinandan13jan) [reply frontend/packages/console-shared/src/components/formik-fields/field-types.ts]: @andrewballantyne I updated and removed the undesired CSS block. I tested many areas like Pipelines, PipelineBuilder, Project Access which makes use of the Component and they all seem to work fine

**Review by andrewballantyne — CHANGES_REQUESTED** [review]
No objection to the changes you made, but I think the height change makes a weird shift in the remove button:

Screen Shot 2021-04-28 at 5 40 18 PM

If we just modify the CSS a tiny bit more, we can achieve a desired experience here. In `frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnField.scss`, if we modify the `&__col--button` and set `top` to `0` and remove `transform` we can align it to the top of the cell so it does not shift down. 

Before/After on a row:
Screen Shot 2021-04-28 at 5 40 18 PM
Screen Shot 2021-04-28 at 5 39 57 PM

Before/After on the larger grouping:
Screen Shot 2021-04-28 at 5 40 25 PM
Screen Shot 2021-04-28 at 5 40 34 PM

**Comment by abhinandan13jan** [comment]
@andrewballantyne Thanks for the input, I've updated accordingly

**Review by andrewballantyne — APPROVED** [review]

**Review by jerolimov — APPROVED** [review]
Tested this locally. Works fine

/lgtm

**Pull request merged**